softdev.bib

```diff
@@ -2698,3 +2698,54 @@ @article{rouvoet20knowing
     url = {https://doi.org/10.1145/3428248},
     doi = {10.1145/3428248},
 }
+
+@inproceedings{demers1989combining,
+    title = "Combining generational and conservative garbage collection: Framework and implementations",
+    author = "Demers, Alan and Weiser, Mark and Hayes, Barry and Boehm, Hans and Bobrow, Daniel and Shenker, Scott",
+    year= 1989,
+    month = Dec,
+    booktitle = "SIGPLAN"
+}
+
+@inproceedings{boehm2003destructors,
+    title = "Destructors, finalizers, and synchronization",
+    author = "Boehm, Hans-J",
+    year = 2003,
+    month = Jan,
+    booktitle = "SIGPLAN"
+}
+
+@article{degenbaev2016idle,
+    title = "Idle time garbage collection scheduling",
+    author = "Degenbaev, Ulan and Eisinger, Jochen and Ernst, Manfred and
+        McIlroy, Ross and Payer, Hannes",
+    year = 2016,
+    month = Jun,
+    journal = "SIGPLAN",
+    volume = 51,
+    number = 6,
+    pages = {570--583}
+}
+
+@article{parkinson2017project,
+    title = "Project snowflake: Non-blocking safe manual memory management in. net",
```

Comment:
This needs to be `Project {Snowflake}: Non-blocking safe manual memory management in {.NET}` (welcome to BibTeX's weird capitalisation rules!).